xdem/coreg.py

```diff
@@ -1063,3 +1081,620 @@ def coregister(reference_raster: Union[str, gu.georaster.Raster], to_be_aligned_
     )
 
     return aligned_raster, error
+
+
+def _transform_to_bounds_and_res(dem: np.ndarray,
```

Comment:
Maybe just pass dem.shape rather than the whole array (in case one does not want to load a large array in the future)

Reply:
Implemented in 2ea683b